mne/time_frequency/tfr.py

```diff
@@ -340,7 +340,8 @@ def _time_frequency(X, Ws, use_fft, decim):
     """Aux of time_frequency for parallel computing over channels
     """
     n_epochs, n_times = X.shape
-    n_times = n_times // decim + bool(n_times % decim)
+    jitter = bool(n_times % decim)
```

Comment:
I actually prefer the way it was before -- `jitter` isn't the right term here, since it's really more about the extra / left-over samples at the end during the decimation step, rather than a jittering process